src/Components/StaticCollapse.tsx

```diff
@@ -30,13 +30,22 @@ export class StaticCollapse extends React.Component<{ open: boolean }> {
     }
 
     this.wrapperRef.addEventListener("transitionend", this.onTransitionEnd)
-    this.wrapperRef.style.height = this.props.open ? "auto" : "0px"
   }
 
   componentDidUpdate() {
     if (!this.wrapperRef) {
       return
     }
+    if (!this.state.hasChanged) {
+      if (this.wrapperRef.style.height === "") {
+        // second render, no update to state, but the style.height value was
+        // unset (See render method) so we need to make sure it's set.
+        // For the remainder of this component's life we control it's
+        // style.height outside of the render cycle
+        this.wrapperRef.style.height = this.props.open ? "auto" : "0px"
+      }
+      return
+    }
```

Comment:
Any reason not to use https://github.com/artsy/palette/tree/master/src/elements/Collapse?

Reply:
yeah check the story for StaticCollapse. Palette's has weird behaviour with multi-line strings.

Reply:
It might be worth debugging the version that's currently in use around our app rather than rolling your own animation code and logic and so on.

Reply:
Given how widely used React Spring is its likely a configuration issue on our end

Reply:
I tried debugging it at the time, but couldn't figure it out and needed to move on.

Reply:
We should avoid adding new components for the sake of expediency if its possible that something just needs to be debugged. Happy to look at it.

Reply:
(Pulled down branch, I see what you mean regarding multi-line text 🤔. Giving it a look)